lib/clientsidescripts.js

```diff
@@ -599,15 +603,36 @@ functions.testForAngular = function(attempts, ng12Hybrid, asyncCallback) {
       asyncCallback(args);
     }, 0);
   };
+  var definitelyNg1 = !!ng12Hybrid;
+  var definitelyNg2 = false;
```

Comment:
We should probably rename this as well, given that angular version thing. What about just `definitelyNg`